### ISSUE TITLE
Method Illuminate\Contracts\Validation\Validator::validateWithBag does not exist

### ISSUE THREAD

**Issue by vanthao03596** [psalm/psalm-plugin-laravel#192]
**Describe the bug**
When run ./vendor/bin/psalm i have this error

**Impacted Versions**
barryvdh/laravel-debugbar             v3.6.2    
barryvdh/laravel-ide-helper           v2.9.1    
fruitcake/laravel-cors                v2.0.4    
laravel/fortify                       v1.7.14   
laravel/framework                     v8.50.0   
laravel/sail                          v1.8.6   
laravel/tinker                        v2.6.1    
psalm/plugin-laravel                  v1.5.0  
spatie/laravel-ray                    1.23.0   
vimeo/psalm                           4.8.1

**Comment by vanthao03596** [comment]
More infomation: Error show in UpdateUserPassword.php and UpdateUserProfileInformation.php from laravel/fortify

**Comment by lptn** [comment]
I don't think this is a Psalm issue this interface rally doesn't have this method. We can try to guess how this interface will be resolved (and will be right probably in 99% cases), but it's even possible to re-bind it in a runtime, so, I prefer to keep it simple and don't try to guess the real type. WDYT, @mr-feek ?
